packages/plugin-data/src/commands/force/data/tree/export.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+
+import { SfdxCommand } from '@salesforce/command';
+import { flags, FlagsConfig } from '@salesforce/command';
+import { Messages, Org } from '@salesforce/core';
+import { ExportApi } from '../../../../lib/data/tree/exportApi';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-data', 'tree.export');
+
+export default class Export extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly requiresProject = false;
```

Comment:
Is this defaulted to false? I thought most things were

Reply:
You're right - thanks!  I'll change it.